experimental/uefi/runtime/src/remote_attestation.rs

```diff
@@ -22,12 +22,31 @@
 extern crate alloc;
 
 use alloc::vec::Vec;
-use anyhow::Context;
-use oak_remote_attestation_sessions::{SessionId, SessionState, SessionTracker};
+use anyhow::{bail, Context};
+use oak_remote_attestation_sessions::{SessionId, SessionState, SessionTracker, SESSION_ID_LENGTH};
 
 /// Number of sessions that will be kept in memory.
 const SESSIONS_CACHE_SIZE: usize = 10000;
 
+fn deserialize_request(msg: Vec<u8>) -> anyhow::Result<(SessionId, Vec<u8>)> {
```

Comment:
nit: a slice (`&[u8]`) can be passed here.

Reply:
Thanks, I'm curious, why would a slice be preferable?

Reply:
A slice is more general, so it would allow calling the function with a slice from an array, a string, a vector etc without having to convert it into a vector first. At the moment I guess this is only used in one place, but in future there might be more uses (e.g. for tests).